### ISSUE TITLE
Readd the .img selection feature

### ISSUE THREAD

**Issue by NightYoshi370** [RocketRobz/TWiLightMenu#372]
With the release of 7.2.0, we saw the addition of loading up SNES and Mega Drive/Genesis using .img files loaded from `sd:\_nds\TWLMenu++\emulators\SNES.img` and `sd:\_nds\TWLMenu++\emulators\GEN.img`. While they involved a bit of work to setup, it had the ability to have roms packed and each .img had a 16/28 MB limit (depending on the console).
The release of 7.3.0 saw the addition of having .smc & .gen files being loaded like other roms (with the brand new .img template method). However, there are a few regressions:

- It has a 3 MB limit, meaning games like Sonic 3 & Knuckles can't be loaded
- It is no longer compatible with lolsnes
- A pack of roms loadable out of the box with no extra work required in TWiLightMenu++ aren't possible anymore (well, until .zip loading is added, that is)
- Switching ROMs without going back to TWiLightMenu++ isn't possible anymore

Now, the third reason doesn't apply too much to me, but the first two does. Therefore, I'd like to have .img loading implemented back into TWiLightMenu++, but not in the exact way as it was done before.
The problem with the way it was done before is that there wasn't really any organization at all. All ROMS had to go in that one .img file and that's it. Therefore, I suggest a new implementation:

Why not have the ability to load any .img file anywhere on the SD card using the actual menu instead of just a button? Then, once that .img file is loaded, it could ask a prompt which emulator it'd like to use and you'd be able to choose. That way, we could have an img file located in the roms folder titled `favorites`, one titled `sonic`, one titled `mario`, and much more.
Having this new system would solve all of the problems I have with the regular loading, and there wouldn't be any regressions of reimplementing this

**Comment by RocketRobz** [comment]
Note that the 3MB limit comes from jEnesisDS, not TWLMenu++.
In jEnesisDS, whenever a 3MB ROM tries to be loaded, the emulator rejects it, and returns to the ROM list.

**Comment by NightYoshi370** [comment]
Really? I thought it was just the .img template limit. My bad

**Comment by chyyran** [comment]
@NightYoshi370 is this still relevant?

**Comment by NightYoshi370** [comment]
Well, it hasn't been implemented yet so I suppose so

**Comment by spellboundtriangle** [comment]
> * It is no longer compatible with lolsnes

Does lolsnes have any advantages over SNEmulDS?


> * A pack of roms loadable out of the box with no extra work required in TWiLightMenu++ aren't possible anymore (well, until .zip loading is added, that is)

What do you mean by this? I'm not sure I understand.

**Comment by NightScript370** [comment]
I have reached the following conclusion after rethinking:
Why should TWiLight Menu++ be responsible for providing a grouping menu for emulators who cannot read SD card files? If we're going to put in work, the work would be better spent on said homebrew.

As such, the only homebrew this would not affect is jEnesisDS, due to it's WEIRD codebase. Since PicoDriveTWL could still be worked on instead, it's best to lay this idea down.

Basically, the gist of this idea was to be able to pass a container (technically .img but you could think .zip) of ROMs to an emulator.
